src/api/middleware/validators/offer.js

```diff
@@ -452,10 +454,30 @@ export const setDefaultValuesCreate = (req, res, next) => {
 };
 
 export const get = useExpressValidators([
-    query("offset", ValidationReasons.DEFAULT)
+    query("lastOfferId", ValidationReasons.DEFAULT)
```

Comment:
Let´s make this generic. In the future we might not want to make the id come from offer ids.

`queryToken` is just a suggestion, you can use something else

```suggestion
    query("queryToken", ValidationReasons.DEFAULT)
```